features/fixtures/mazerunner/jvm-scenarios/src/main/java/com/bugsnag/android/mazerunner/scenarios/JvmAnrMinimalFixtureScenario.kt

```diff
@@ -5,23 +5,24 @@ import com.bugsnag.android.Configuration
 import android.content.Context
 import android.os.Looper
 import android.os.Handler
+import com.bugsnag.android.Bugsnag
 
 /**
  * Stops the app from responding for a time period
  */
-internal class AppNotRespondingDisabledNdkScenario(config: Configuration,
-                                                   context: Context) : Scenario(config, context) {
+internal class JvmAnrMinimalFixtureScenario(config: Configuration,
+                                            context: Context) : Scenario(config, context) {
     init {
         config.autoTrackSessions = false
         config.enabledErrorTypes.anrs = true
-        config.enabledErrorTypes.ndkCrashes = false
     }
 
     override fun run() {
         super.run()
+        Bugsnag.notify(generateException())
         val main = Handler(Looper.getMainLooper())
         main.postDelayed(Runnable {
-            Thread.sleep(50000) // FOREVER
+            Thread.sleep(50000)
```

Comment:
Can you inline-comment the rationale for choosing `Thread.sleep` over a `while(true)`? I think that would help future selves read the test.